Towny/src/main/java/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -804,7 +806,7 @@ public void onPlayerTeleport(PlayerTeleportEvent event) {
 				event.setCancelled(true);
 				return;
 			}
-			if (!TownySettings.JailAllowsTeleportItems() && (event.getCause() == TeleportCause.ENDER_PEARL || event.getCause() == TeleportCause.CHORUS_FRUIT)) {
+			if (!TownySettings.JailAllowsTeleportItems() && (event.getCause() == TeleportCause.ENDER_PEARL || event.getCause() == TeleportCause.CONSUMABLE_EFFECT)) {
```

Comment:
I intentionally hadn't replaced this one yet since CONSUMABLE_EFFECT doesn't exist in 1.19, which is currently the minimum required version. We'll need something that uses consumable effect if it exists and otherwise fall back to chorus fruit.